provider/tags.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+package gcp
+
+import (
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	shim "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim"
+)
+
+func fixTagNames(prov *tfbridge.ProviderInfo) {
+	// Recursively applies the tag fix
```

Comment:
I think we may want a couple words here to indicate what the tag/label fix is and why we're doing it.